qa/os/src/test/java/org/elasticsearch/packaging/util/ServerUtils.java

```diff
@@ -325,6 +325,28 @@ public static void possiblyDisableSecurityFeatures(Installation installation) th
         }
     }
 
+    /**
+     * Explicitly disables security features
+     */
+    public static void disableSecurityFeatures(Installation installation) throws IOException {
```

Comment:
We still need to do this as many test expect security to be disabled. I plan on takiing care of those in a follow up ( to keep it self-contained) and this method can be removed